### PR TITLE
added solution for jv-storage

### PR DESCRIPTION
**Pull request by Rosinskyi**

### CODE REVIEW

**Comment by Rosinskyi** [comment]
i'm trying to make it without HashMap

File: src/main/java/core/basesyntax/impl/StorageImpl.java [hunk 1]
```diff
@@ -2,15 +2,24 @@
...
 import core.basesyntax.Storage;
 
-public class StorageImpl<K, V> implements Storage<K,V> {
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    private K key;
+    private V value;
+    private Map<K, V> map = new HashMap<>();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/impl/StorageImpl.java]: we should create it without hash map

File: src/main/java/core/basesyntax/impl/StorageImpl.java [hunk 2]
```diff
@@ -2,15 +2,34 @@
 
 import core.basesyntax.Storage;
 
-public class StorageImpl<K, V> implements Storage<K,V> {
+public class StorageImpl<K, V> implements Storage<K, V> {
+    private int index = 0;
+    private int capacity = 16;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/impl/StorageImpl.java]:
```suggestion
    private static final int CAPACITY = 16;
```

File: src/main/java/core/basesyntax/impl/StorageImpl.java [hunk 3]
```diff
@@ -2,15 +2,34 @@
...
+        }
+        keys[index] = key;
+        values[index] = value;
+        index++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < index; i++) {
+            System.out.println(values[i]);
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/impl/StorageImpl.java]: try to remove this line from here

**Pull request closed without merging**